README.md

```diff
@@ -1 +1,77 @@
 # Physlr: Construct a Physical Map from Linked Reads
+
+Physlr constructs a *de novo* physical map using linked reads from 10X Genomics or stLFR. This physical map can then be used to scaffold an existing assembly to yield chromosomal level contiguity.
+
+# Dependencies
+
+* [ntCard](https://github.com/bcgsc/ntCard)
+* [ntHits](https://github.com/bcgsc/ntHits)
+* GCC 5 or newer with OpenMP
+* Python 3.5 or newer and the following packages
+    * [community](https://python-louvain.readthedocs.io/en/latest/api.html)
+    * [networkx](https://networkx.github.io/)
+    * [numpy](https://numpy.org/)
+    * [scipy](https://www.scipy.org/)
+    * [sklearn](https://scikit-learn.org/stable/)
+    * [tqdm](https://tqdm.github.io/)
+
+Additionally, we recommend using pypy3 over regular python3 for speed.
+
+
+## Optional dependencies
+
+- [pigz](https://zlib.net/pigz/) for parallel gzip
+- [zsh](https://sourceforge.net/projects/zsh/) for reporting time and memory usage
+
+
+# Compiling Physlr from source
+
+```
+pip3 install --user git+https://github.com/bcgsc/physlr
+cd src
+make
+```
+
+# Running Physlr
+
+## Generating Physlr Physical Map with stLFR reads
+
+To construct a physical map, you need linked reads from 10X Genomics or stLFR. In addition, to visualize the correctness and contiguity of the physical map, you need a reference genome.
+In this example, the linked reads and reference genome are called `linkedReads.fq.gz` and `reference.fa`, respectively. The linked reads are from stLFR so we specify `minimizer_overlap=stLFR` to use the default value for stLFR reads.
```

Comment:
Rename `linkedReads.fq.gz` to `linkedreads.fq.gz`. Capital letters in filenames aren't typical and can cause confusion.